pkg/e2e/infra/utils.go

```diff
@@ -319,3 +319,32 @@ func getScaleClient() (scale.ScalesGetter, error) {
 	}
 	return scaleClient, nil
 }
+
+// nodesSnapShot logs the state of all the nodes in the cluster
+func nodesSnapShot(client runtimeclient.Client) error {
```

Comment:
Rename this to capture its intent; it was only when I read the comment that I see it is related to gathering logs. `snapshotNodeLogs()`?